controller-api/src/main/java/com/yahoo/vespa/hosted/controller/api/integration/zone/ZoneRegistry.java

```diff
@@ -75,4 +75,7 @@
     /** Returns a URL used to request support from the Vespa team. */
     URI supportUrl();
 
+    /** Returns a URL used to generate flashy badges from strings. */
+    URI badgeUrl();
```

Comment:
Most of these URLs aren't actually zone-related. I think we should pull them out and group them together elsewhere soon (e.g. config).

Reply:
Yes.

Reply:
They're typically _system_ related. Perhaps we should allow a richer view of a system from the controller (and likewise for a _zone_, but that's just vaguely related)?